From 33e0f4a41b5718cf826ac80c24c830a0f7913887 Mon Sep 17 00:00:00 2001
From: hongjiasen <hongjiasen@allwinnertech.com>
Date: Thu, 7 Sep 2023 15:18:09 +0800
Subject: [PATCH] fix pcm open no such file or directory.

Signed-off-by: hongjiasen <hongjiasen@allwinnertech.com>
---
 src/conf/alsa.conf | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/src/conf/alsa.conf b/src/conf/alsa.conf
index e65bf2c..937884e 100644
--- a/src/conf/alsa.conf
+++ b/src/conf/alsa.conf
@@ -10,7 +10,13 @@
 		files [
 			"/var/lib/alsa/conf.d"
 			"/usr/etc/alsa/conf.d"
-			"/etc/alsa/conf.d"
+			{
+				@func concat
+				strings [
+					{ @func datadir }
+					"/alsa.conf.d/"
+				]
+			}
 			"/etc/asound.conf|||/usr/etc/asound.conf"
 			"~/.asoundrc"
 			{
-- 
2.29.0

